packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -48,10 +51,19 @@ export const Alert: React.FunctionComponent<AlertProps> = ({
   title,
   children = '',
   className = '',
+  truncateTitle = 0,
   ouiaId,
   ouiaSafe = true,
   ...props
 }: AlertProps) => {
+  const titleRef = React.useRef(null);
+  React.useEffect(() => {
+    if (!titleRef.current || !truncateTitle) {
+      return;
+    }
+    titleRef.current.style.setProperty(maxLines.name, truncateTitle.toString());
```

Comment:
Also might be better to limit just the `title` from props in `getHeadingContent` if it's a string. Otherwise consumer has to do the truncation.

Reply:
I am not sure what you meant. But I cannot do that in `getHeadingContent` as it is wrapping `title` in `React.Fragment`:

```
    <React.Fragment>
      <span className={css(accessibleStyles.screenReader)}>{variantLabel}</span>
      {title}
    </React.Fragment>
```

I guess I misunderstood you. Can you give an example?

Reply:
Here you could trim the `title` if I am reading the code correctly. But as I said, that's my preference to trim the string instead of css rule. But both are doable!